RFunction.R

```diff
@@ -1,444 +1,446 @@
-library('move2')
-library('geodist')
-library('lubridate')
-library('lutz')
-library('sf')
-library('terra')
+library("move2")
```

Comment:
Most changes were just my automated linter

Reply:
what is "linter"?

Reply:
It's a package that checks the syntax/style of the code automatically to adhere to the language's style guide

I use tidyverse's: https://www.tidyverse.org/blog/2017/12/styler-1.0.0/